### ISSUE TITLE
Update() only uses the cached value, even if there's no cached value

### ISSUE THREAD

**Issue by Kampfkarren** [Kampfkarren/Roblox#57]
Easy fix, but a backwards incompatible one. Will have to check if anyone is relying on this behavior, though I can't imagine it.

**Comment by ElliottLMz** [comment]
Seeing that this issue has been open for nearly 4 months, can there be a potential fix for this?

I strongly doubt that anyone is relying on this behavior (especially seeing that this issue lacks any objections or comments). This is also listed as unintentional behavior in the docs, further supporting that people _shouldn't be relying on this behavior._

**Comment by Kampfkarren** [comment]
I just haven't gotten around to fixing this yet is all.

**Comment by DimitarBogdanov** [comment]
Any updates on this?

**Comment by Kampfkarren** [comment]
Nope, my projects move with me, meaning that if I don't need functionality, then someone else needs to put it in. PR's are welcome.

**Comment by DimitarBogdanov** [comment]
I'll go ahead and try to patch when I get back from holiday. Happy new year!

**Comment by AmateurJavaCoder** [comment]
Any news on this? Did you manage to add it, DimitarBogdanov?

**Comment by DimitarBogdanov** [comment]
> Any news on this? Did you manage to add it, DimitarBogdanov?

I totally forgot about the existence of this, I haven't used DS2 for a long time now. Sorry for the cliffhanger there!

**Comment by AmateurJavaCoder** [comment]
The suspense was killing me!

Thanks for responding anyway xD
